tests/integration/pilot/vm/vm_test.go

```diff
@@ -96,38 +71,44 @@ spec:
 					ServicePort:  8090,
 				},
 			}
-			var pod, vm echo.Instance
-			echoboot.NewBuilderOrFail(t, ctx).
-				With(&vm, echo.Config{
-					Service:    "vm",
-					Namespace:  ns,
-					Ports:      ports,
-					Pilot:      p,
-					DeployAsVM: true,
-				}).
-				With(&pod, echo.Config{
-					Service:   "pod",
-					Namespace: ns,
-					Ports:     ports,
-					Pilot:     p,
-				}).
-				BuildOrFail(t)
+			for i, vmImage := range vmImages {
+				ctx.NewSubTest(vmImage).
+					Run(func(ctx framework.TestContext) {
+						var pod, vm echo.Instance
```

Comment:
we can use on pod and just create N VMs probably?

Reply:
Understood. Will update.